arrayfire/array.py

```diff
@@ -1312,6 +1328,44 @@ def __array__(self):
         safe_call(backend.get().af_get_data_ptr(c_void_ptr_t(res.ctypes.data), self.arr))
         return res
 
+    def to_ndarray(self, output=None):
+        """
+        Parameters
+        -----------
+        output: optional: numpy. default: None
+
+        Returns
+        ----------
+        If output is None: Constructs a numpy.array from arrayfire.Array
+        If output is not None: copies content of af.array into numpy array.
+
+        Note
+        ------
+
+        - An exception is thrown when output is not None and it is not contiguous.
+        - When output is None, The returned array is in fortran contiguous order.
```

Comment:
nit: I prefer column major order rather than Fortran contiguous order.

Reply:
@umar456 That's the notation numpy uses.